src/tools/build-manifest/src/main.rs

```diff
@@ -97,6 +97,7 @@ static TARGETS: &[&str] = &[
     "i686-unknown-linux-gnu",
     "i686-unknown-linux-musl",
     "i686-unknown-uefi",
+    "loongarch64-unknown-linux-gnu",
```

Comment:
I don't think this is needed for tier 3 targets, as we don't ship any artifacts.

Reply:
Thank you, it has been removed from the patch set.